main.go

```diff
@@ -341,6 +341,7 @@ func renderRTEManifests(rteManifests rtemanifests.Manifests, namespace string, i
 		return mf, err
 	}
 	_ = rteupdate.DaemonSetUserImageSettings(mf.DaemonSet, "", imageSpec, images.NullPolicy)
+	//nolint:errcheck
 	rteupdate.DaemonSetPauseContainerSettings(mf.DaemonSet)
```

Comment:
we should actually bubble up the error here (both calls)

Reply:
done.